src/services/searchPatient.ts

```diff
@@ -36,7 +36,7 @@ export const searchPatient = async (
     let searchByIdentifier = ''
     let filterByService = ''
     if (groupId) {
-      const services = (await getServices(groupId)).filter((service) => undefined !== service.id)
+      const services = (await getOrganizations(groupId)).filter((service) => undefined !== service.id)
```

Comment:
I don't get this.. what's a group id here ? if we're talking about a cohort group, how this would work if `getOrganizations` accept only Organization ids?

Reply:
don't understand neither

Reply:
I've fixed this part in the previous PR. We can remove this during merging